pyicloud/services/photos.py

```diff
@@ -0,0 +1,286 @@
+import sys
+import json
+import urllib
+
+from datetime import datetime
+from base64 import b64decode
+from bitstring import ConstBitStream
+from pyicloud.exceptions import PyiCloudBinaryFeedParseError
+
+
+class PhotosService(object):
+    """ The 'Photos' iCloud service."""
+
+    def __init__(self, service_root, session, params):
+        self.session = session
+        self.params = dict(params)
+
+        self.prepostfetch = 200
+
+        self._service_root = service_root
+        self._service_endpoint = '%s/ph' % self._service_root
+
+        request = self.session.get(
+            '%s/startup' % self._service_endpoint,
+            params=self.params
+        )
+        response = request.json()
+        self.params.update({
+            'syncToken': response['syncToken'],
```

Comment:
Oh, I didn't really expect you to spend time reproducing it, by the way, @torarnv; I would've thought a `request.raise_for_status()` would've been totally sufficient, but I've just tried and have seen that you've already added a proper error message there now.